### PR TITLE
Fix test case always pass issue

### PR DESCRIPTION
**Pull request by guessi**
**Issue #, if available:** n/a

**Description of changes:**

Inconsistent `EXIT_CODE`, `exit_code` will make test case always pass.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Testing Done**
- n/a

*[See this guide for recommended testing for PRs.](../doc/CONTRIBUTING.md#testing-changes) Some tests may not apply. Completing tests and providing additional validation steps are not required, but it is recommended and may reduce review time and time to merge.*

### CODE REVIEW

File: test/cases/mount-bpf-fs.sh [hunk 1]
```diff
@@ -61,7 +61,7 @@ EXIT_CODE=0
 /etc/eks/bootstrap.sh \
   --b64-cluster-ca dGVzdA== \
   --apiserver-endpoint http://my-api-endpoint \
-  test || exit_code=$?
+  test || EXIT_CODE=$?
 if [[ ${EXIT_CODE} -ne 0 ]]; then
```
Comment (guessi) [inline test/cases/mount-bpf-fs.sh:65]: `EXIT_CODE` will always be `0` as it have been reset to `0` at Line-60.

File: test/cases/mount-bpf-fs.sh [hunk 2]
```diff
@@ -84,7 +84,7 @@ EXIT_CODE=0
...
```
Comment (guessi) [inline test/cases/mount-bpf-fs.sh:88]: `EXIT_CODE` will always be `0` as it have been reset to `0` at Line-83.

**Comment by cartermckinnon** [comment]
Thanks -- some test cases use `exit_code`, so this was a copy-paste mishap.

**Review by cartermckinnon — APPROVED** [review]

**Comment by cartermckinnon** [comment]
Tests still pass, nice. 😄

**Pull request merged**